doc/introduction/overview.rst

```diff
@@ -0,0 +1,72 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _introduction/overview:
+
+********
+Overview
+********
+
+Prerequisites
+=============
+
+To be able to get the most out of this document, we assume you have basic knowledge of:
+
+ * What ftrack(https://ftrack.com) are and what it does.
+ * :term:`Python`, specifically the ftrack Python API.
+ * Source code management, specifically Git.
+ * :term:`DCC` applications, i.e. Maya, Nuke.
+ * ftrack :term:`Connect` 2 Package, installation - app launch - publish and load.
+
+
+Key design elements and tradeoffs
+=================================
+
+Connect 2 comes with a new framework, designed to be a starting point for developers
```

Comment:
The pre-build of Connect 2 comes with the new framework or they have to download it throw the Plugin manager?

Reply:
Removed Connect 2 here aswell